Ctfl-at-chapter2.md

```
@@ -0,0 +1,663 @@
+# ISTQB® CTFL-AT Chapter 2: アジャイルテストの基本原則
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)